docs/sdk/dev/initialize-and-run-workflows.md

```diff
@@ -62,12 +56,7 @@ There are the following commands for the workflow:
   spryker-sdk sdk:init:project --workflow={workflowName} --workflow={workflowName}
   ```
   ---
-{% info_block infoBox "Initialising workflows" %}
-
-If you initialize workflows for the project, you can use only these workflows.
-
-{% endinfo_block %}
-  
+  **Note:** If you initialize workflows for the project, you can use only these workflows.
```

Comment:
{% info_block warningBox "Workflow Initialization" %}

If you initialize workflows for the project, you can use only these workflows.

 {% endinfo_block %}